scripts/st2_bootstrap.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+install_dependencies() {
+  sudo apt-get update
+  sudo apt-get install -y mongodb-server rabbitmq-server postgresql
+}
+
+setup_repositories() {
+  wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
+  echo "deb https://dl.bintray.com/stackstorm/trusty_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
+  sudo apt-get update
+}
+
+install_stackstorm_components() {
+  sudo apt-get update
+  sudo apt-get install -y st2 st2mistral
+}
+
+setup_mistral_database() {
+  cat << EHD | sudo -u postgres psql
+CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
+CREATE DATABASE mistral OWNER mistral;
+EHD
+
+  # Setup Mistral DB tables, etc.
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+  # Register mistral actions
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+}
```

Comment:
idea (soft) How about `echo "Ignore the  novaclient.client.Client warnings above, it's OK."`